src/components/PeopleFilters.tsx

```diff
@@ -27,66 +60,57 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {filterByCentury.map(century => (
+              <Link
+                key={century}
+                data-cy="century"
+                className={classNames(
+                  'button mr-1',
+                  { 'is-info': centuries.includes(century) },
+                )}
+                to={{
+                  search: getSearchWith(searchParams, {
+                    centuries: centuries.includes(century)
+                      ? centuries.filter(c => c !== century)
+                      : [...centuries, century],
+                  }),
+                }}
+              >
```

Comment:
Nice job using `getSearchWith`! But we can do even better.
Check out `SearchLink.tsx`, it contains a wrapper for `Link` that passes your params to query.

The `SearchLink` already includes `useSearchParams` hook, so you would not need to rely on it in every component where you update your query seach params.